kge/model/kg2e.py

```diff
@@ -0,0 +1,105 @@
+import torch
+from kge import Config, Dataset
+from kge.model.kge_model import KgeModel, RelationalScorer
+from kge.job import Job
+from torch.nn import functional as F
+
+
+def score_kl(mean, ent_cov, rel_cov):
+    return -(
+        torch.sum((1 / rel_cov) * ent_cov, dim=-1)
+        + torch.sum(mean * mean * (1 / rel_cov), dim=-1)
+        - torch.sum(torch.log(ent_cov), dim=-1)
+        + torch.sum(torch.log(rel_cov), dim=-1)
+    )
+
+
+def score_el(mean, ent_cov, rel_cov):
+    return -(
+        torch.sum(mean * mean * (1 / (ent_cov + rel_cov)), dim=-1)
+        + torch.sum(torch.log(ent_cov + rel_cov), dim=-1)
+    )
```

Comment:
Overall this looks good. But I have a question regarding the scoring function. In the paper, the scoring function starts with 1/2,  but yours starts with -1. Is this right?

Reply:
Yes, in the original paper, they derive the score to be 1/2. As the scores are only used to compare two triples, I think it shouldn't matter, if they are scaled by a constant factor. 
The times -1 is used because it turns the score reported in the paper, which indicates similarity, into a dissimilarity score. (compare to [Wang2017](https://persagen.com/files/misc/Wang2017Knowledge.pdf), Table 1)